README.md

```diff
@@ -62,99 +108,83 @@ exports.process = function process(msg, cfg, snapshot) { };
 
-A `fixtures.json` file has a root fixtures object, and then one or more defined fixtures. Below is an example file:
+### Examples
 
-````json
-{
-    "fixtures":{
-        "success":{
-            "msg":{
-                "headers":{},
-                "body":{}
-            },
-            "cfg":{
-                "calendarId":"{{GOOGLE_CALENDAR_ID}}",
-                "oauth":{
-                    "expires_in":3600,
-                    "token_type":"Bearer",
-                    "refresh_token":"{{GOOGLE_REFRESH_TOKEN}}",
-                    "access_token":"{{GOOGLE_ACCESS_TOKEN}}"
-                }
-            }
-        }
-    }
-}
-````
+`> elasticio cmp:process -x success -a lookupObject`
 
-The content of the file is a JSON object with a single key ``fixtures`` which contains named fixtures. Each fixture defines arguments to be passed to component's _process_ function: a message and configuration objects. 
+### Limitations
+None at the moment
 
-The only fixture in example above is named _success_. Please note that the  component's configuration may contain sensible data, such as API keys or OAuth tokens. Such data must not be placed inside fixture files because you will push them to your version control system, such as GitHub. Instead they should be replaced by variables using the [Handlebars](http://handlebarsjs.com/) syntax:
+## cmp:exec
 
-````
-"access_token":"{{google_access_token}}"
-````
+`cmp:exec` will allow you to run any exported function from an action/trigger that may be used on the platform, or `verifyCredentials` for a given component. For example, this allows you to run the functions `getMetaModel`, and other Select View functions. It can be run from within a component directory, or from outside by providing a path.
 
-The variable values will be taken from the `.env` file stored in your component, in the same folder as `component.json`.
+The CLI tool assumes that exported functions and `verifyCredentials` will have the following signatures:
 
-```bash
-GOOGLE_CALENDAR_ID=fubar@acme.org
-GOOGLE_REFRESH_TOKEN=very-secret-refresh-token
-GOOGLE_ACCESS_TOKEN=very-secret-access-token
+```javascript
+exports.verify = async function verify(cfg, optional callback); // found in verifyCredentials.js
+exports.getMetaModel = function getMetaModel(cfg);
+exports.selectViewFunctions = function selectView(cfg);
+exports.process = function process(msg, cfg, snapshot);
 ```
 
-If you need to add global variables that are used within your functions, for example `ELASTICIO_EXEC_ID`, add them in a file located at `/test/.globalEnv` and they will be exported and added to `process.env` automatically.
+`getMetaModel` and any Select View function should take the `cfg` as first parameter, as should `verify`. For this function, running `process` will not run it with the startup/shutdown functions, and will run only the selected method in isolation. This can be beneficial for testing purposes.
```

Comment:
```suggestion
`verify`, `getMetaModel`, and any Select View function should take the `cfg` as first parameter. For this function, running `process` will not run it with the startup/shutdown functions, and will run only the selected method in isolation. This can be beneficial for testing purposes.
```